lib/link_expansion/hashed_edition.rb

```diff
@@ -0,0 +1,37 @@
+module LinkExpansion::HashedEdition
```

Comment:
@kevindew questioned whether the `HashedEdition` module shouldn't be a utility class with a class methods, at this point I decided to rename it to `EditionHash` as the original name felt a bit ambiguous.